src/auro-header.js

```diff
@@ -78,6 +87,15 @@ export class AuroHeader extends LitElement {
   spacingApplied(size) {
     switch (size) {
       case 'none': return `util_stackMarginNone--${this.margin}`
+      case '25': return `util_stackMargin25--${this.margin}`
+      case '50': return `util_stackMargin50--${this.margin}`
+      case '100': return `util_stackMargin100--${this.margin}`
+      case '150': return `util_stackMargin150--${this.margin}`
+      case '200': return `util_stackMargin200--${this.margin}`
+      case '300': return `util_stackMargin300--${this.margin}`
+      case '400': return `util_stackMargin400--${this.margin}`
+      case '600': return `util_stackMarginX600--${this.margin}`
+      case '800': return `util_stackMargin800--${this.margin}`
       case 'xxxs': return `util_stackMarginXxxs--${this.margin}`
       case 'xxs': return `util_stackMarginXxs--${this.margin}`
       case 'xs': return `util_stackMarginXs--${this.margin}`
```

Comment:
Same comment as above about deprecation.

Reply:
@blackfalcon , could you please explain as this code was changed by you?

Reply:
See previous comment.